fmc/src/flow/mod.rs

```diff
@@ -0,0 +1,62 @@
+/*++
+
+Licensed under the Apache-2.0 license.
+
+File Name:
+
+    mod.rs
+
+Abstract:
+
+    File contains the top level dispatch of various RT Flows.
+
+--*/
+
+mod cold_reset;
+mod unknown_reset;
+mod update_reset;
+mod warm_reset;
+
+pub mod dice;
+mod rt_alias;
+
+use crate::flow::dice::{DiceInput, DiceLayer};
+use crate::flow::rt_alias::RtAliasLayer;
+
+use crate::fmc_env::FmcEnv;
+use crate::HandOff;
+use caliptra_drivers::CaliptraResult;
+use caliptra_drivers::ResetReason;
+
+/// Extract DiceInput from handoff information.
+pub fn try_from_hand_off(_env: &FmcEnv, _hand_off: &mut HandOff) -> Option<DiceInput> {
+    None
+}
+
+/// Execute FMC Flows based on reset resason
+///
+/// # Arguments
+///
+/// * `env` - FMC Environment
+pub fn run(env: &FmcEnv, hand_off: &mut HandOff) -> CaliptraResult<()> {
+    // Placeholder.
+    if let Some(input) = try_from_hand_off(env, hand_off) {
+        let _ = RtAliasLayer::derive(env, &input);
+    } else {
+        // TODO : handle error
```

Comment:
Should this error just be passed up the call stack and dealt with in main?

Reply:
I am making flow our top level module for FMC logic and entry_point  (main) just a Rust entry point, but I am open to other suggestions :-)

Reply:
works for me